tutorials/text/pgf.py

```diff
@@ -56,6 +56,30 @@
 
 .. _pgf-rcfonts:
 
+
+Multi-Page PDF Files
```

Comment:
add entries to doc/api/howto_faq (which mentions the old pdfpages) and examples/misc/multipage_pdf?

Reply:
done